app/workers/initial_tagging_import.rb

```diff
@@ -1,11 +1,12 @@
+# TODO: put this safely into BulkTagging module
```

Comment:
We need to keep the old class, in cases there are in-flight jobs during deploy. That seemed too much for this PR.